pinns/src/pinns.c

```diff
@@ -83,12 +86,15 @@ int main(int argc, char **argv) {
       unshare_flags |= CLONE_NEWCGROUP;
       bind_cgroup = true;
       num_unshares++;
+      break;
```

Comment:
This seems like a real fix. If so, it deserves a separate commit.

Reply:
good point, done

Reply:
done